runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ReferencesSection/ReferencesSection.module.scss

```diff
@@ -0,0 +1,170 @@
+@import '#src-landing/scss/main.scss';
+
+.root {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    max-width: 1600px;
+    margin: 0 auto;
+    padding: 120px $padding-3;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.rectangleContainer {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    width: 100%;
+}
+
+.rectangle {
+    width: 85%;
+    max-width: 1364px;
+    max-height: 608px;
+    background: $palette-white;
+    box-shadow: 0px 6px 16px rgba(0, 0, 0, 0.2);
+    border-radius: 120px 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.smallRectangle {
+    position: absolute;
+    width: 120px;
+    height: 128px;
+    background: #FCE8B1;
+    border-radius: 20px 20px 150px 20px;
+    right: -20px;
+    bottom: 0;
+}
+
+
+.title {
+    width: 85%;
+    text-align: left;
+    margin-bottom: $margin-4;
+}
+
+.contentWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    width: 100%;
+}
+
+.logosContainer {
+    width: 85%;
+    max-width: 1364px;
+    padding: $padding-2;
+    overflow: hidden;
+    position: relative;
+}
+
+.logos {
+    display: flex;
+    justify-content: space-around;
+    margin-top: $margin-4;
+    gap: 20px;
+    transition: transform 0.3s ease-in-out;
+}
+
+.logo {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: $padding-5 $padding-0;
+    gap: 10px;
+    min-width: 260px;
+    min-height: 160px;
+    border-radius: 20px;
+    box-sizing: border-box;
+    cursor: pointer;
+
+    &.selected {
+        box-shadow: 0px 6px 16px rgba(0, 0, 0, 0.2);
+        border: 1px solid #C7C7C7;
+    }
+
+    img {
+        border-radius: 20px;
+        height: 70px;
+        object-fit: contain;
+    }
+}
+
+.dotsContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: $margin-1;
+    margin-top: $margin-2;
```

Comment:
if possible use padding instead of margins - the margin overlapping can occur sometimes